tecken/libmarkus.py

```diff
@@ -43,53 +38,12 @@ def set_up_markus(backends, hostname, debug=False):
         # In local dev and test environments, we want the RegisteredMetricsFilter to
         # raise exceptions when metrics are used incorrectly.
         metrics_filter = RegisteredMetricsFilter(
-            registered_metrics=TECKEN_METRICS, raise_error=True
+            registered_metrics=STATSD_METRICS, raise_error=True
         )
         METRICS.filters.append(metrics_filter)
 
     if hostname:
         # Define host tag here instead of in the backend so it shows up in tests
         METRICS.filters.append(AddTagFilter(f"host:{hostname}"))
 
-    _IS_MARKUS_SETUP = True
-
-
-class LogAllMetricsKeys(BackendBase):  # pragma: no cover
```

Comment:
We don't need this since we only allow documented metrics to be used via the `RegisteredMetricsFilter`.